.github/workflows/monodocs_build.yml

```
@@ -0,0 +1,52 @@
+name: Docs Build
```

From https://github.com/flyteorg/flytekit/actions/runs/7147108206 I think there is a concurrency conflict with https://github.com/flyteorg/flytekit/blob/f0d4f1357a770b66dc8344ab97b8b41d19eca9df/.github/workflows/docs_build.yml#L1

I think changing the name should work:

```suggestion
name: Monodocs Build
```

---

@cosmicBboy The name change has to be at the workflow level (not the job level).

---

done!